src/main/java/callofcactus/io/DatabaseManager.java

```diff
@@ -248,6 +181,13 @@ public Connection getConnection() {
 		return connection;
 	}
 
+	public void addAccount(String username, String password) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Document empty method body](https://www.codacy.com/app/guushamm/Call-of-Cactus/file/1388792630/issues/source?bid=2979575&fileBranchId=2980831#l184)